src/Microsoft.AspNetCore.Server.Kestrel/KestrelServerLimits.cs

```diff
@@ -108,5 +111,27 @@ public int MaxRequestHeaderCount
                 _maxRequestHeaderCount = value;
             }
         }
+
+        /// <summary>
+        /// Gets or sets the keep-alive timeout, in seconds.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to 5 seconds.
+        /// </remarks>
+        public int KeepAliveTimeout
```

Comment:
`(int)Math.Ceiling(timespan.TotalSeconds)` then add to remarks "rounds up to second"?